backend/graminsta/post/views.py

```diff
@@ -178,19 +183,58 @@ def get(self, request):
             "fan_count": fan_count})
 
 
-class UserView(APIView):
+class CommentView(APIView):
     """
-    A class based view to list all the users.
+    A class based view to manage comments.
     """
     @staticmethod
-    def get(request):
-        """Gets all the users and if they are followed by
-            the request user.
+    def get(request, post_id):
+        """
+        Gets the given post's comments
 
         Parameters
         ----------
         request: GET request
+        post_id: int
 
+        Returns
+        -------
+        response: json format
+            Comments that the post owns
+        """
+        comments = get_all_comments(post_id=post_id)
+        return Response(CommentSerializer(comments, many=True).data)
+
+    @staticmethod
+    def post(request):
+        """
+        Creates a new comment
+
+        Parameters
+        ----------
+        request: json format
+            Data containing all comments
+        """
+        user = request.user
+        comment = request.data.get('comment')
+        post_id = request.data.get("post_id")
+        comment = add_comment(post_id, user, comment)
+
+        comments = get_all_comments(post_id=post_id)
```

Comment:
Hmm, I think the response should only contains one Comment instead of returning all for this post. Though it can reduce  the burden for frontend but it's not a good practice.